src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
```

Comment:
In this case, forEach returns the value, but it shouldn't do it, must be replaced with 
men.forEach((m) => { sum += m.died - m.born) }
Anyway it's better to use a reduce method for sum operations 
const totalAge = men.reduce((acc, m) => (
    acc + m.died - m.born
  ), 0);